src/domUpdates.js

```diff
@@ -22,34 +22,39 @@ const roomTypeButton = document.getElementById('selectTypeButton');
 
 const greeting = document.querySelector('.greeting');
 
-const toBookDisplay= document.querySelector('.to-book-display')
+const toBookDisplay = document.querySelector('.to-book-display')
+
+const roomTypeContainer = document.querySelector('.room-type-container-js')
 
 
 
 //FUNCTIONS
 
-function show(elements) {
+
+
+const show = (elements) => {
   elements.forEach(element => {
+    console.log(element)
     element.classList.remove('hidden');
   });
 }
 
-function hide(elements) {
+const hide = (elements) => {
   elements.forEach(element => {
     element.classList.add('hidden');
   });
 }
 
-
 //DOM UPDATES OBJECT
 
 let domUpdates = {
 
-  populateCustomerInfo(currentCustomer, roomsData) {
+  welcomeUser() {
```

Comment:
Took these lines of code out of the what-is-now populateCustomerBookings function so that it was only populating the necessary info (the bookings) once I'd fetched the data again after a post. Didn't need to be re-running code for the username, etc.